Sources/XcbeautifyLib/Pattern.swift

```diff
@@ -41,7 +41,7 @@ enum Pattern: String {
 
     /// Regular expression captured groups:
     /// $1 = file
-    case codesign = #"CodeSign\s((?:\ |[^ ])*)$"#
+    case codesign = #"CodeSign\s(((?!.framework/Versions/A)(?:\ |[^ ]))*)$"#
```

Comment:
since now, whichever patter matches first is winning, we need to make sure that `codesign` pattern will not 'eat'  `codesignFramework` pattern